### ISSUE TITLE
Scroll to any element not just Actionable

### ISSUE THREAD

**Issue by alan-morey** [salesforce/utam-java#171]
### Is your feature request related to a problem? Please describe

I've defined a cancel button in my page object as follows:
```json
...
            {
                "name": "cancelButton",
                "selector": {
                    "css": "lightning-button:nth-of-type(1)"
                },
                "type": "utam-lightning/pageObjects/button",
                "public": true
            },
...
```

I'm trying to click the `lightning-button` on my page object, the button element implements the `utam.core.element.Clickable` interface, so it has the `click` method exposed. However, in the UI, the button element is being rendered within modal and the modal body has scrollbars so the buttons are not currently visible, as they are overlapped by the modal container. When the `click()` method is called while the button is not visible an exception is raised: `ElementClickInterceptedException`:

```
element click intercepted: Element <button class="slds-button slds-button_neutral" type="button">...</button> is not clickable at point (968, 786). Other element would receive the click: <div class="modal-container slds-modal__container" data-aura-rendered-by="982:0">...</div>
```

While debugging, I manually scrolled the modal body in the browser window and executed the `click()` again and then it worked.

I was thinking I could compose a custom click method in my utam page object to scroll to the button element before clicking it.

However, currently only elements that implement the `utam.core.element.Actionable` will have the methods `scrollToCenter` and `scrollToTop` exposed.

### Describe the solution you'd like

I think the `Clickable` interface should also have these `scrollTo*` methods?

Or maybe those methods should be pulled up the class hierarchy up to `utam.core.element.BasicElement` or even `utam.core.framework.UtamBase` so that any element could be scrolled to, not just those elements that are actionable / clickable?

### Additional context

utam-java v1.1.3
salesforce-pageobjects v2.2.0

**Comment by olivier-martin-sf** [comment]
Thank you for opening this issue @alan-morey. 

You are absolutely right, the `lightning-button` page object should be declared as both `actionable` and `clickable` and it has been updated internally already. We will publish the updated version in our next release of the salesforce-pageobjects JAR for the spring"23 Salesforce release.

We are going to discuss with the team about your proposal of pulling up the scrolling method higher in the class hierarchy and will give you feedback once we came up with a decision.

Thanks a lot for the detailed and very helpful feedback!
